device-types/Supermicro/SuperServer-6018U-TR4T-plus.yaml

```diff
@@ -0,0 +1,31 @@
+---
+manufacturer: Supermicro
+model: SuperServer 6018U-TR4T+
+slug: supermicro-sys-6018u-tr4t-plus
+part_number: SYS-6018U-TR4T+
+u_height: 1
+is_full_depth: true
+weight: 18.6
+weight_unit: kg
+airflow: front-to-rear
+comments: 'Specification: [https://www.supermicro.com/en/products/system/1U/6018/SYS-6018U-TR4T_.cfm](https://www.supermicro.com/en/products/system/1U/6018/SYS-6018U-TR4T_.cfm)'
+power-ports:
+  - name: PSU1
+    type: iec-60320-c14
+    maximum_draw: 750
+  - name: PSU2
+    type: iec-60320-c14
+    maximum_draw: 750
```

Comment:
These PSUs appear to be modular. If that is true, these should be defined as `module-bays` and the associated PSUs should be created in the module-types folder